packages/spectrum/src/layouts/GroupLayout.tsx

```diff
@@ -64,7 +64,7 @@ export const GroupLayoutRenderer: FunctionComponent<RendererProps & VanillaRende
             {group.label}
           </legend> : ''
       }
-      {renderChildren(group, schema, childClassNames, path)}
+      {renderChildren(group, schema, /*childClassNames*/ {}, path)}
     </fieldset>
   );
 };
```

Comment:
Das GroupLayout gehe ich als nächstes an, habe es hier mal rundimentär and die Changes im `renderChildren` angepasst.